tests/test_isamples_metadata.py

```diff
@@ -263,12 +266,13 @@ def _get_record_with_id(record_id: str) -> typing.Dict:
 
 @pytest.mark.parametrize("isamples_path", SMITHSONIAN_test_values)
 def test_smithsonian_dicts_equal(isamples_path):
-    id_piece = re.search(r"-([^-]+)-test", isamples_path).group(1)
-    source_dict = _get_record_with_id(id_piece)
-    # create the transformer from the specified row in the source .csv
-    transformer = SmithsonianTransformer(source_dict)
-    transformed_to_isamples_record = transformer.transform()
-    _assert_transformed_dictionary(isamples_path, transformed_to_isamples_record)
+    with patch.object(ModelServerClient, "make_smithsonian_sampled_feature_request", return_value="foo"):
```

Comment:
This syntax is saying "if you hit `make_smithsonian_sampled_feature_request` in the `ModelServerClient` class, return "foo" instead of invoking that code.